### PR TITLE
Update google-g-suite-apps-audit.md

### PR DESCRIPTION
**Pull request by kparekh010**
## Purpose of this pull request

This pull request (PR) is to have a single source of truth between the CSE docs and send data docs. 

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/ingestion/google-g-suite-apps-audit.md [hunk 1]
```diff
@@ -28,18 +21,7 @@ In this step, you configure an G Suite Apps Audit Source to collect G Suite log
...
-1. Navigate to the Hosted Collector where you want to create the source.
-1. On the **Collectors** page, click **Add Source** next to the Hosted Collector.
-1. Select **G Suite Apps Audit.** 
-1. The page refreshes.  
-    ![gsuite-apps-audit-source.png](/img/cse/gsuite-apps-audit-source.png)
-1. **Name**. Enter a name for the source. 
-1. **Description**. (Optional) 
-1. **Application**. Select the G Suite app you wish to collect using this source. Steps may be repeated for each G Suite app you want to collect from.
-1. **Source Category**. Enter a string to tag the output collected from the source. The string that you supply will be saved in a metadata field called `_sourceCategory.`
+1. To configure G suite source, see [Configure a G-suite Apps Audit Source] (/docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source/#configuring-a-g-suite-apps-auditsource).
```
Comment (JainM6) [inline docs/cse/ingestion/google-g-suite-apps-audit.md:24]:
```suggestion
1. To configure G suite source, see [Configure a G-suite Apps Audit Source](/docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source/#configuring-a-g-suite-apps-auditsource).
```

Comment (JainM6) [inline docs/cse/ingestion/google-g-suite-apps-audit.md:24]:
```suggestion
1. To configure a G suite source, see [Configure a G-suite Apps Audit Source] (/docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source/#configuring-a-g-suite-apps-auditsource).
```

**Review by JainM6 — DISMISSED** [review]
Minor formatting changes

File: docs/cse/ingestion/google-g-suite-apps-audit.md [hunk 2]
```diff
@@ -28,18 +21,7 @@ In this step, you configure an G Suite Apps Audit Source to collect G Suite log
...
-1. Navigate to the Hosted Collector where you want to create the source.
-1. On the **Collectors** page, click **Add Source** next to the Hosted Collector.
-1. Select **G Suite Apps Audit.** 
-1. The page refreshes.  
-    ![gsuite-apps-audit-source.png](/img/cse/gsuite-apps-audit-source.png)
-1. **Name**. Enter a name for the source. 
-1. **Description**. (Optional) 
-1. **Application**. Select the G Suite app you wish to collect using this source. Steps may be repeated for each G Suite app you want to collect from.
-1. **Source Category**. Enter a string to tag the output collected from the source. The string that you supply will be saved in a metadata field called `_sourceCategory.`
+1. To configure G suite source, see [Configure a G-suite Apps Audit Source](/docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source/#configuring-a-g-suite-apps-auditsource).
```
Comment (kimsauce) [inline docs/cse/ingestion/google-g-suite-apps-audit.md:24]:
```suggestion
1. To configure G Suite source, see [Configure a G Suite Apps Audit Source](/docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source/#configuring-a-g-suite-apps-auditsource).
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**